packages/fxa-settings/public/index.html

```diff
@@ -12,6 +12,11 @@
     content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=2,user-scalable=yes" />
   <meta name="fxa-config" content="__SERVER_CONFIG__" />
 
+  <link rel="preconnect" href="__GQL_URL__">
+  <link rel="preconnect" href="__AUTH_URL__">
+  <link rel="preconnect" href="__OAUTH_URL__">
```

Comment:
I don't think we need this one.  I'm not sure how frequently Settings directly connects to the profile server URL, if at all.